config.js

```diff
@@ -2,7 +2,24 @@ module.exports = {
   "flags": {
     "injectTestUrls": false,
     "itemsExpandable": false,
-    "itemsSwipable": false
+    "itemsSwipable": false,
+  },
+
+  "userFlags": {
+    "sortByDistance": {
+      value: false, // default
+      title: 'Sort by distance',
```

Comment:
So far as we are not thinking on l10n the app is ok, but when we do it we will need to revisit this.

Anyway, so far for the experiment is perfect.